### PR TITLE
SpConfig TEPP

### PR DESCRIPTION
**Pull request by potatoqualitee**

### CODE REVIEW

**Comment by potatoqualitee** [comment]
hey @wsmelton - i havent tested these, do you have time? I needed them and noticed the configs were missing so I added them back

**Comment by wsmelton** [comment]
Sure pulling it down now.

File: functions/Get-DbaSpConfigure.ps1 [hunk 1]
```diff
@@ -55,55 +55,58 @@ function Get-DbaSpConfigure {
 		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
 		[Alias("ServerInstance", "SqlServer", "SqlServers")]
 		[DbaInstanceParameter[]]$SqlInstance,
-		[System.Management.Automation.PSCredential]$SqlCredential
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[object[]]$Config
```
Comment (wsmelton) [inline functions/Get-DbaSpConfigure.ps1]: I think this needs to match the property you have to return: `ConfigName` and not just `Config`. The object output for the value is set as `ConfigName`.

Comment (wsmelton) [inline functions/Get-DbaSpConfigure.ps1]: Go ahead and add `Silent` switch. Line 68 looks to be the only change required for `Write-Message`.

**Comment by wsmelton** [comment]
Added few notes. The reference for naming the TEPP parameter ConfigName applies to the Set command as well. I forgot to add the review on that command as well.

**Comment by wsmelton** [comment]
Adjusted the parameter name while you were sleeping 👿 

If you disagree you can always put it back. I did add `Config` as an alias on both commands, updated help, and examples.

**Comment by potatoqualitee** [comment]
LOL, thank youuuuu. I've been sleeping since about 7pm yesterday

**Pull request merged**